### ISSUE TITLE
Face gender sample code

### ISSUE THREAD

**Issue by cftang0827** [ageitgey/face_recognition#449]
* face_recognition version: latest
* Python version: 3.6
* Operating System: Ubuntu/Mac

### Description

Hi, thanks for ageitgey and dlib's excellent API.

 I am doing a face gender classifier by using face_encodings as a feature extractor and scikit-learn SVM. Here is my code. The code and pretrained model is provided, feel free to use it. 

### What I Did

```
https://github.com/cftang0827/gender_classifier_dlib_transfer_learning
```

**Comment by ageitgey** [comment]
Thanks for sharing your code. This kind of feature extractor approach could be useful to people doing all sorts of different things.

**Comment by MLDSBigGuy** [comment]
Thank you @cftang0827 :) Thats so sweet :) May i know the accuracy of your classifier ?

**Comment by cftang0827** [comment]
Hi, here is the evaluation metric

                 precision    recall  f1-score   support

              f       0.90      0.99      0.94       159
              m       0.99      0.91      0.95       189

     avg / total       0.95      0.95      0.95       348

And you can also see it on my github, due to the lack of Asian face data, so the accuracy is not so high compare with other DNN approach.

I think the accuracy could be improved by using neuron network, or using image augmentation to gain more data.

Thank you :)
